packages/fast-jss-utilities/README.md

```diff
@@ -92,3 +92,23 @@ const styles = {
     }
 };
 ```
+
+### applyAcrylic
+The `applyAcrylic` function can be used to create a partially transparent texture for the background of a specific element or UI surface.
+
+```ts
+import { applyAcrylic } from "@microsoft/fast-jss-utilities";
+
+const acrylicConfig: IAcrylicConfig = {

```

Comment:
can we remove the typescript here and use js?

Reply:
all of these are all ts, so if we make that change I'd prefer to do it in a new PR

Reply:
sounds good